src/manifest.rs

```diff
@@ -26,6 +26,9 @@ static DEFAULT_MANIFEST: &str = r#"
 pub struct Manifest {
     pub tools: BTreeMap<ToolAlias, ToolId>,
 
+    #[serde(skip)]
+    pub token: Option<String>,
+
```

Comment:
We should store authentication tokens in a separate place than the manifest file. Otherwise, users may accidentally commit them to version control or expose them in other places. We should add a new config file to store authentication, and explicitly label this as a GitHub PAT.

Reply:
I thought about this for a bit, how's `.aftmantoken`?

Reply:
We already have a `.aftman` directory. Adding an `auth.toml` file to that directory would be a good idea.

Reply:
`~/.aftman` is global though?
As I was thinking though that you could have a different token for different orgs sometimes...

Reply:
Authentication with a service like GitHub is global. You should have one GitHub account, and accessing different organizations does not make it so you need multiple tokens. All private repositories can be accessed via a single PAT for your account.